syntaxes/hubl.json

```diff
@@ -115,7 +119,7 @@
               "name": "variable.other.hubl.test"
             }
           },
-          "match": "\\s*\\b(is)\\s+([a-zA-Z_][a-zA-Z0-9_]*)\\b"
+          "match": "\\s*\\b(is)(?!\\s+not)\\s+([a-zA-Z_][a-zA-Z0-9_]*)\\b"
```

Comment:
Added a non-capturing group for `is` so that this doesn't match `x is not y`, only `x is y`. We need to keep this group around because the alternative (adding `is` to the group above) would match independent `is` tokens incorrectly, so `x not is y` would get highlighted as an operator which seems undesirable (really we should error on that but outside the scope of this change).